code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -832,3 +928,130 @@
 	active = FALSE
 	owner.client?.view_size.resetToDefault()
 	COOLDOWN_START(src, last_toggle, 4 SECONDS)
+
+/datum/action/cooldown/spell/aoe/repulse/frenzied_roar
+	name = "Famished Roar"
+	desc = "An AOE roar spell that sends all near by people flying."
+	background_icon_state = "bg_heretic"
+	overlay_icon_state = "bg_heretic_border"
+	button_icon = 'icons/mob/actions/humble/actions_humble.dmi'
+	button_icon_state = "void_magnet"
+	sound = 'yogstation/sound/magic/demented_outburst_scream.ogg'
+
+	school = SCHOOL_FORBIDDEN
+	invocation = "GR' RO'AR"
+	invocation_type = INVOCATION_SHOUT
+	spell_requirements = NONE
+	aoe_radius = 3
+
+	cooldown_time = 60 SECONDS
+
+/datum/action/cooldown/spell/pointed/projectile/eldritchbolt
+	name = "Eldritch Bolt"
+	desc = "Fire a bolt of Eldritch energy. It will jump between targets, but can't knock them down."
```

Comment:
instead of making it like this make it a subtype of the original bolt spell then revar it

Reply:
done